eva_submission/eload_submission.py

```diff
@@ -219,11 +222,12 @@ def detect_metadata_attributes(self):
 
     def find_genome(self):
         scientific_name = self.eload_cfg.query('submission', 'scientific_name')
-        assembly_accession = self.eload_cfg.query('submission', 'assembly_accession')
-        if scientific_name and assembly_accession:
-            assembly_fasta_path, assembly_report_path = get_genome_fasta_and_report(scientific_name, assembly_accession)
-            self.eload_cfg.set('submission', 'assembly_fasta', value=assembly_fasta_path)
-            self.eload_cfg.set('submission', 'assembly_report', value=assembly_report_path)
+        reference_accession = self.eload_cfg.query('submission', 'assembly_accession')
+        if scientific_name and reference_accession:
+                assembly_fasta_path, assembly_report_path = get_reference_fasta_and_report(scientific_name, reference_accession)
+                self.eload_cfg.set('submission', 'assembly_fasta', value=assembly_fasta_path)
+                self.eload_cfg.set('submission', 'assembly_report', value=assembly_report_path)
```

Comment:
Might be good to log a warning if the assembly report is missing, since for now at least later parts of the automation might break.